pypfopt/cla.py

```diff
@@ -445,15 +445,15 @@ def set_weights(self, _):
         # Overrides parent method since set_weights does nothing.
         raise NotImplementedError("set_weights does nothing for CLA")
 
-    def portfolio_performance(self, verbose=False, risk_free_rate=0.02):
+    def portfolio_performance(self, risk_free_rate=0.02, verbose=False):
```

Comment:
oops, I didn't necessarily mean to reorder these. I suppose it makes sense? idk, is there a Python style guide for this type of thing?

Reply:
Robert says in the [docs](https://pyportfolioopt.readthedocs.io/en/latest/Contributing.html) that he prefers Python code in [Black](https://github.com/psf/black) style. I haven't looked at it personally, but maybe you can see if it's relevant to the keyword ordering?